tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -54,9 +53,14 @@ time pip install --user virtualenv
 time pip install --user --upgrade Mako six tox setuptools twisted pyyaml pyjwt cryptography requests
 export PYTHONPATH=/Library/Python/3.4/site-packages
 
-# Install Python 3.7
+# Install Python 3.7 and Python 3.8
 time curl -O https://www.python.org/ftp/python/3.7.0/python-3.7.0-macosx10.9.pkg
+time curl -O https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg
+echo "ae0717a02efea3b0eb34aadc680dc498 python-3.7.0-macosx10.9.pkg" > /tmp/python_installer_checksum.md5
+echo "f5f9ae9f416170c6355cab7256bb75b5 python-3.8.0-macosx10.9.pkg" >> /tmp/python_installer_checksum.md5
```

Comment:
I suppose this would have already bitten us if this were a problem, but are we in control of the version of Mac OS that's being used here? If so, this is sort of fragile. If not, this version may be *quite* old. In fact, the operating system isn't even called "OS X" anymore.

This version looks like it's from 2013. We should probably add a TODO and file an issue for updating this. At some point, this version will probably no longer be adequate.

Reply:
TODO issue created https://github.com/grpc/grpc/issues/21304